airbyte-cdk/python/airbyte_cdk/test/utils/reading.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional, List, Mapping, Any
+
+from airbyte_cdk import AbstractSource
+from airbyte_cdk.test.catalog_builder import CatalogBuilder
+from airbyte_protocol.models import AirbyteStateMessage, ConfiguredAirbyteCatalog, SyncMode
+from airbyte_cdk.test.entrypoint_wrapper import EntrypointOutput, read
+
+
+def catalog(stream_name: str, sync_mode: SyncMode) -> ConfiguredAirbyteCatalog:
```

Comment:
It makes a ton of sense to me to bring these two methods in the CDK since they're reimplemented by all connectors!

Reply:
I am personally skeptic about this one because of the "brittleness" I mentioned [here](https://github.com/airbytehq/airbyte/pull/43338/files#r1711444572). I feel like the builder is as readable as this method and I don't know what is the benefit of this method

Reply:
I'd be fine with a builder. what I like about the current approach is we get to remove the duplicated `_read` static methods https://github.com/airbytehq/airbyte/pull/43338/files#diff-22af5fa8ee65b48a3c85cc96dca9cd0900df9a5d3378ee9da0cad9ba84952d9eL20-L21

Reply:
These functions are what many tests use, I just moved them. These are provided in a common location for people to use them when they don't need the full builder or an `__init__()` with [default arguments](https://stackoverflow.com/questions/11977279/builder-pattern-equivalent-in-python). It's difficult to generalize everything, but these little functions come handy in many cases, if they are not handy, they can fall back to the builder.